CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 This file is used to list changes made in each version of the libarchive cookbook.
 
-## 2.0.0 (2024-060-06)
+## 1.2.0 (2024-06-10)
```

Comment:
I don't fully understand this version bump. Why are we moving from 1.0.0 to 1.2.0 (rather than 1.1.0)? It also looks like we are updating a previous changelog entry rather than adding a new one.